userguide/virtualmachines.rst

```diff
@@ -663,107 +663,88 @@ Start the Docker Host
-Host. The card shows the :guilabel:`Com Port` for the Docker Host. In
-this example, :literal:`/dev/nmdm12B` is used.
-
-Use an SSH client to connect to the %brand% server. Remember this also
-requires the :ref:`SSH` service to be running. Depending on the %brand%
-system configuration, it might also require changes to the
-:guilabel:`SSH` service settings, like setting
-:guilabel:`Login as Root with Password`.
-
-At the %brand% console prompt, connect to the Rancher Host with
-`cu <https://www.freebsd.org/cgi/man.cgi?query=cu>`__, replacing
-:samp:`{/dev/nmdm12B}` with the value from the Docker Host
-:guilabel:`Com Port`:
+:menuselection:`Virtual Machines`
+and click
+|ui-options| :menuselection:`--> Start`
+to boot the Docker Host.
 
-.. code-block:: none
+A Docker Host can take some time to boot. Connect to the Serial Shell
+during the startup process to view the activity of the Docker Host.
 
-   cu -l /dev/nmdm12B -s 9600
+.. tip:: A :command:`cu -l` command displays as the shell connects to
+   the Docker Host Com Port. This command can also be used to connect
+   to the Docker Host during an SSH session into the %brand%
+   system. The :samp:`nmdm{XY}` value is different for each started
+   Docker Host.
 
 
-If the terminal does not show a :literal:`rancher login:` prompt,
-press :kbd:`Enter`. The Docker Host can take some time to start and
-display the login prompt.
+When a message about :literal:`RancherOS` starting appears and the shell
+stops posting new messages, press :kbd:`Enter` to see the
+:literal:`ClientHost login:` text and continue to log in.
 
 
 .. _Installing and Configuring the Rancher Server:
 
 Installing and Configuring the Rancher Server
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Go to
-:menuselection:`Virtual Machines`
-and locate the card for the Docker Host. Start the Host and click
-:guilabel:`Connect` and :guilabel:`Serial` to open the Host Serial shell.
-
-Continuing to set up the Rancher Host is done from the command line.
-Enter *rancher* as the username, press :kbd:`Enter`, then enter either
-the default password *docker* or the custom password created by editing
-the raw file. Press :kbd:`Enter` again. After logging in, a
-:literal:`[rancher@ClientHost ~]$` prompt is displayed.
+In the serial shell, type :literal:`rancher` for the username and press
+:kbd:`Enter`. Enter either the default password :literal:`docker` or the
+custom password created by editing the raw file and press :kbd:`Enter`.
+A :literal:`[rancher@ClientHost ~]$` prompt will appear.
 
 Ensure Rancher has functional networking and can :command:`ping` an
-outside website. Adjust the VM
-:ref:`Network Interface <vms-network-interface>` and reboot the VM
-if necessary.
-
-Download and install the Rancher system with this command:
+outside website.
 
 .. code-block:: none
 
-   sudo docker run -d --restart=unless-stopped -p 8080:8080 rancher/server
+   [rancher@ClientHost ~]$ ping -c 5 8.8.8.8
```

Comment:
8.8.8.8 is a magic number here, and is not going to tell the user what is going on.  Are we using a bare IP address because we don't know if DNS is working?